### ISSUE TITLE
Low serum reverse T3 concentration

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11224]
**Preferred term label:**

**Synonyms**
Decreased serum rT3 

**Definition (free text, please give PubMed ID)**
PMID20301682: Males with AHDS have pathognomonic thyroid test results (Figure 1) including the following:
• High serum 3,3',5-triiodothyronine (usually free T3) concentration and low serum 3,3',5'-triiodothyronine 
(reverse T3, or rT3) concentration

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Allan-Herndon-Dudley syndrome ORPHA:59

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001334
- label: Decreased circulating reverse T3 concentration
- parents: [Abnormal circulating thyroid hormone concentration (HP:0031508)](https://hpo.jax.org/app/browse/term/HP:0031508)
- pmid: 
- def: Concentration of reverse T3 (3,3',5'-triiodothyronine or rT3) in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11224
- synonyms:
